calculator.js

```diff
@@ -0,0 +1,37 @@
+console.log("Hello, I am a caluculator.");
+console.log("You can use the command 'calculator.(add/subtract/multiply/divide/exponent/remainder)(numberYouWant, otherNumberYouWant)' to get calculations.");
+
+var caluculator = {add: function (x, y){
+  var numAdd = x + y;
+  console.log(x + " + " + y + " = " + numAdd);
+  return numAdd;
+}, subtract: function (x, y){
+  var numSubtract = x - y;
+  console.log(x + " - " + y + " = " + numSubtract);
+  return numSubtract;
+}, multiply: function (x, y){
+  var numMultiply = x * y;
+  console.log(x + " * " + y + " = " + numMultiply);
+  return numMultiply;
+}, divide: function (x, y){
+  var numDivide = x - y;
+  console.log(x + " / " + y + " = " + numDivide);
+  return numDivide;
+}, exponent: function (x, y){
```

Comment:
Dig the hand-rolled approach! Worth checking out `Math.pow()`, which handles this all in one command.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/pow